models/cow_protocol/ethereum/cow_protocol_ethereum_trades.sql

```diff
@@ -245,5 +248,5 @@ valued_trades as (
 select *,
   -- Relative surplus (in %) is the difference between limit price and executed price as a ratio of the limit price.
   -- Absolute surplus (in USD) is relative surplus multiplied with the value of the trade
-  fill_proportion * usd_value * (atoms_bought * limit_sell_amount - atoms_sold * limit_buy_amount) / (atoms_bought * limit_sell_amount) as surplus_usd
+  usd_value * (atoms_bought * limit_sell_amount - atoms_sold * limit_buy_amount) / (atoms_bought * limit_sell_amount) as surplus_usd
```

Comment:
cc @harisang -- this here

Reply:
Why are we reverting this?

Reply:
Did you read the PR description?

Reply:
Cool. Thanks!

Reply:
Out of curiosity, `usd_value` is equal to the value of the trade? So, for partial fills, this still works because it again describes the value of what was traded?

If so, how is this measured? Is the price of the sell token used, the price of the buy token, an average of the two maybe?

Reply:
We use the max of the (sell_value, buy_value)